src/azure-cli/azure/cli/command_modules/security/commands.py

```diff
@@ -485,15 +491,22 @@ def load_command_table(self, _):
     with self.command_group('security automation-action-logic-app',
                             security_automations_sdk,
                             client_factory=cf_security_automations) as g:
+
         g.custom_command('create', 'create_security_automation_action_logic_app')
     with self.command_group('security automation-action-event-hub',
                             security_automations_sdk,
                             client_factory=cf_security_automations) as g:
+
         g.custom_command('create', 'create_security_automation_action_event_hub')
     with self.command_group('security automation-action-workspace',
                             security_automations_sdk,
                             client_factory=cf_security_automations) as g:
         g.custom_command('create', 'create_security_automation_action_workspace')
 
+    with self.command_group('security security_solutions',
```

Comment:
We never use `_` to seperate command name. Please use '-' instead.

```suggestion
    with self.command_group('security security-solutions',
```